vllm/entrypoints/llm.py

```diff
@@ -202,11 +207,12 @@ def _run_engine(self, use_tqdm: bool, profiling: bool = False) -> List[RequestOu
                     outputs.append(output)
                     if use_tqdm:
                         pbar.update(1)
-            if profiling and is_hpu():
-                htorch.core.mark_step()
```

Comment:
What is the purpose of removing `mark_step` here?

Reply:
A better place for setting graph boundries is in 'llm_engine.step()' . Besides that, previously, that mark_step was only triggered when profiling. If it was really needed it should've been called unconditionally.